common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -52,10 +52,12 @@ Notch Filter Control Types
 #. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 2 (RPM Sensor 1) or 5(RPM Sensor2). **RPM sensor based**, where an external :ref:`RPM sensor <common-rpm>` is used to determine the motor frequency and hence primary vibration source's frequency for the notch. Often used in Traditional Helicopters (See :ref:`Helicopters<common-imu-notch-filtering-helicopter-setup>`) using the ArduPilot Head Speed Governor feature. See :ref:`RPM Sensor<common-rpm-based-notch>` for further setup instructions.
 #. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 3. **ESC Telemetry based**, where the ESC provides motor RPM information which is used to set the center frequency. This can also be used for the forward motor in fixed wing flight, if the forward motor(s) ESCs report RPM. This requires that your ESCs are configured correctly to support BLHeli telemetry via :ref:`a serial port<blheli32-esc-telemetry>`. See :ref:`ESC Telemetry<common-esc-telem-based-notch>` for further setup instructions. If :ref:`INS_HNTCH_OPTS<INS_HNTCH_OPTS>`, or :ref:`INS_HNTC2_OPTS<INS_HNTC2_OPTS>` if the second set of notches is enabled, has bit 1 set, then a set of notches for each motor will be created, tracking its RPM telemetry, otherwise, the average frequency of all motors will set the center frequency.
-#. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 4. **In-Flight FFT based**, where a running FFT is done in flight to determine the primary noise frequency and adjust the notch's center frequency to match. This probably the best mode if the autopilot is capable of running this feature. It requires that the autopilot firmware supports it (see :ref:`common-limited_firmware` for GyroFFT feature) and has sufficient cpu power (F7/H7 autopilots). This mode also works on fixed wing only Planes. See :ref:`In-Flight FFT <common-imu-fft>` for further setup instructions.
+#. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 4. **In-Flight FFT based**, where a running FFT is done in flight to determine the primary noise frequency and adjust the notch's center frequency to match. This probably the best mode if the autopilot is capable of running this feature. It requires that the autopilot firmware supports it (see :ref:`common-limited_firmware` for GyroFFT feature) and has sufficient cpu power (F7/H7 autopilots). See :ref:`In-Flight FFT <common-imu-fft>` for further setup instructions.
+
+.. note:: If using the **Throttle based** control, and the autopilot supports **In-Flight FFT based**, you can temporarily use the latter to determine the center frequency parameters via log analysis for the **Throttle based** control setup since it is sometimes preferable, due to its faster response controlling the notch frequency, than **In-Flight FFT based** for aerobatic quadcopters, etc.
```

Comment:
should say "without log analysis" - its preferable because its more convenient than log analysis rather than more accurate

Reply:
so running FFT sets the HNTCH_FREQ used by throttle notch and it persists? what about _REF and _BW? detail the steps for param setup of throtlle notch but using fft first

Reply:
@andyp1per ?

Reply:
Not quite, its this option - https://youtu.be/9nlcdKQgRPU

So normal FFT will just drive the notch as you expect. You can also use RCx_OPTION 162 to get the throttle notch setup using the FFT. I feel like you documented this somewhere because its in this PR - https://github.com/ArduPilot/ardupilot/pull/20230